packages/hash/frontend/src/pages/type-editor/index.page.tsx

```diff
@@ -45,27 +46,30 @@ const ExampleUsage = ({ accountId }: { accountId: string }) => {
 
   const createPropertyType = useCallback(() => {
     void (async () => {
-      await functions
-        .createPropertyType({
-          data: {
-            propertyType: {
-              kind: "propertyType",
-              $id: propertyUri,
-              title: "Name",
-              oneOf: [
-                {
-                  $ref: "https://blockprotocol.org/@blockprotocol/types/data-type/text/v/1",
-                },
-              ],
+      if (isVersionedUri(propertyUri)) {
+        // now propertyUri is typed as VersionedUri from the type system
```

Comment:
I think this comment is a little confusing if I'm honest

Reply:
If you're trying to indicate that this is using methods from the type system then maybe rephrase as something like "use the type guard from the package"

Reply:
I think it's an unnecessary comment anyway, I'll remove it 250b06ad829bdf3d603d00be256a59d31b02c396